--- cabextract-1.1/configure.ac~	2004-10-15 23:05:09.000000000 +0300
+++ cabextract-1.1/configure.ac	2006-04-22 20:45:51.574146230 +0300
@@ -9,7 +9,6 @@
 # Checks for programs.
 AC_PROG_CC
 AC_PROG_INSTALL
-AC_PROG_RANLIB
 AC_EXEEXT
 
 # Checks for header files.
--- cabextract-1.2/Makefile.am.orig	2006-09-21 01:10:40.000000000 +0200
+++ cabextract-1.2/Makefile.am	2006-09-27 12:03:40.649292250 +0200
@@ -8,17 +8,8 @@
 
 man_MANS =		doc/cabextract.1
 
-AM_CPPFLAGS = 		-I$(srcdir)/mspack -DMSPACK_NO_DEFAULT_SYSTEM
-noinst_LIBRARIES =	libmspack.a
-libmspack_a_SOURCES =	mspack/mspack.h \
-			mspack/system.h mspack/system.c \
-			mspack/cab.h mspack/cabd.c \
-			mspack/lzx.h mspack/lzxd.c \
-			mspack/mszip.h mspack/mszipd.c \
-			mspack/qtm.h mspack/qtmd.c
-
 bin_PROGRAMS =		cabextract
 noinst_PROGRAMS =	src/cabinfo
 cabextract_SOURCES =	src/cabextract.c md5.h md5.c
-cabextract_LDADD =	libmspack.a @LIBOBJS@
+cabextract_LDADD =	-lmspack @LIBOBJS@
 
