diff -uNr cabextract-0.6.orig/configure.in cabextract-0.6/configure.in
--- cabextract-0.6.orig/configure.in	Sun Aug 11 21:18:36 2002
+++ cabextract-0.6/configure.in	Sun Aug 25 22:20:03 2002
@@ -6,6 +6,7 @@
 dnl Checks for programs.
 AC_PROG_CC
 AC_PROG_INSTALL
+AC_PROG_LIBTOOL
 AC_EXEEXT
 
 dnl Checks for libraries.
@@ -29,6 +30,6 @@
 AC_REPLACE_FUNCS(mktime)
 
 dnl check for getopt in standard library
-AC_CHECK_FUNCS(getopt_long , , [LIBOBJS="$LIBOBJS getopt.o getopt1.o"] ) 
+AC_CHECK_FUNCS(getopt_long , , [AC_LIBOBJ(getopt); AC_LIBOBJ(getopt1)] )
 
 AC_OUTPUT(Makefile cabextract.spec)
